newsfragments/2634.feature.rst

```diff
@@ -0,0 +1,2 @@
+Adds a new CLI command to show past and present staking rewards, "stake rewards show".
+Moves "stake collect-reward" to "stake rewards withdraw" command.
```

Comment:
I think this second line entry can be put in a `2634.removal.rst` newsfragment (really as a deprecation not a removal). Although @KPrasch may feel differently 😜

Reply:
I think it's fair because this is a breaking change.